store/conf/attrs/zimbra-attrs.xml

```diff
@@ -10571,12 +10571,12 @@ TODO: delete them permanently from here
   </desc>
 </attr>
 
-<attr id="4138" name="zimbraFeatureExternalEmailWarningEnabled" type="boolean" cardinality="single" optionalIn="globalConfig,domain" flags="domainInherited,domainAdminModifiable" since="10.1.9">
+<attr id="4138" name="zimbraFeatureExternalEmailWarningEnabled" type="boolean" cardinality="single" optionalIn="globalConfig,domain" flags="domainInherited,domainAdminModifiable" since="10.1.12">
```

Comment:
Any reason to change since value?

Reply:
@silentsakky This ldap is not yet released